scripts/binary/util/packages.ts

```diff
@@ -1,16 +1,21 @@
 import _ from 'lodash'
 import fs from 'fs-extra'
 import path from 'path'
-// we wrap glob to handle EMFILE error
+import glob from 'glob'
 import la from 'lazy-ass'
 import check from 'check-more-types'
 import execa from 'execa'
 import debugLib from 'debug'
 
-import externalUtils, { globby } from './3rd-party'
-
 const debug = debugLib('cypress:binary')
 
+type globAsyncType = (p: string, o?: object) => Promise<string[]>;
+const globAsync: globAsyncType = async (pattern, options = {}) => {
+  return new Promise((resolve, reject) => {
+    glob(pattern, options, (err, files) => err === null ? resolve(files) : reject(err))
+  })
+}
+
```

Comment:
nit:  `Bluebird.promisify` or Node's builtin `util.promisify` would be more concise and you'd get the types for free, as long as `glob` has types